test/unit/edition/translatable_test.rb

```diff
@@ -36,8 +36,17 @@ class Edition::TranslatableTest < ActiveSupport::TestCase
     assert_not create(:news_article).locale_can_be_changed?
   end
 
+  test "locale_can_be_changed? returns true for a new DocumentCollection" do
+    assert DocumentCollection.new.locale_can_be_changed?
+  end
+
+  test "locale_can_be_changed? returns true for an existing DocumentCollection" do
+    doc_collection = create(:document_collection)
+    assert doc_collection.locale_can_be_changed?
+  end
```

Comment:
These could become one test:
```
test "locale_can_be_changed? returns true for existing and new DocumentCollections" do
  new_doc_collection = DocumentCollection.new
  existing_doc_collection = create(:document_collection)
  assert [new_doc_collection, existing_doc_collection].all?(&:locale_can_be_changed?)
end
```
The benefit would be that when we quickly scan the code we'd be less likely to skip one of the two. Also, it makes it extremely clear that we expect the same behaviour for both new and existing document collections.